docs/data/routes/guides/multisite-guide/implementation-patterns/en.md

```diff
@@ -0,0 +1,64 @@
+---
+name: implementation-patterns
+routeTemplate: ./data/component-templates/guide.yml
+title: Implementation Patterns
+---
+
+## Multisite in JSS
+With the out-of-the-box configuration, each JSS app creates one site definition and generates an independent code bundle. So to support multisite scenarios, it is necessary to create and deploy multiple JSS apps.
+
+If you deploy 3 JSS apps to the same Sitecore instance, this will create 3 independent sites, and 3 independent webpack bundles on the filesystem that would be structured like this:
+
+```
+<sitecore installation root>
+  /dist
+    /site 1
+    /site 2
+    /site 3
+```
+
+After deploying all the apps and switching to Sitecore-first workflow, there is no technical limitation preventing you from re-organizing item and references from the Sitecore side to be more multisite compatible. For example, sharing templates between the sites, settings up placeholder settings in such a way as to share components between sites, etc. The challenge is that from the code side, each app only sees it’s own code, so achieving actual sharing capabilities requires some customization.
+
+### Microsites under the same domain
+Micro-sites can be separate JSS apps and stay under the same domain. In their site definitions, they need to have different virtual paths and different site names.
+
+You need to ensure that the sc_site parameter is always populated in calls to Layout Service to use this technique. 
```

Comment:
It is also possible to override the Layout Service ItemResolver to resolve a virtual folder from the 'item' query string parameter. We need to do that once and all JSS apps don't need to care of the sc_site parameter anymore.

Reply:
https://www.brimit.com/blog/how-to-resolve-a-virtual-directory-on-a-jss-sitecore-website

Reply:
Thanks! I added a note for on-prem users